frontend/src/app/toolbar/toolbar.component.ts

```diff
@@ -1,20 +1,52 @@
-import { Component, Input } from '@angular/core';
-import { Router } from '@angular/router';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges,
+} from '@angular/core';
 import { MenuService } from '../menu.service';
 
 @Component({
   selector: 'app-toolbar',
   templateUrl: './toolbar.component.html',
-  styleUrls: ['./toolbar.component.scss']
+  styleUrls: ['./toolbar.component.scss'],
 })
-export class ToolbarComponent {
+export class ToolbarComponent implements OnChanges {
   @Input()
   title?: string;
 
   @Input()
   backUrl?: string;
 
-  constructor(public menuService: MenuService, private router: Router) {
+  @Output()
+  changeChoreographyName = new EventEmitter<string>();
```

Comment:
**issue:** is the same toolbar used in acrobatics too? 🤔

Reply:
Nice catch! Changed the name to changeTitle and added a boolean input to determine whether the title should be editable or not.